src/main/java/levi9/UserService/service/impl/UserServiceImpl.java

```diff
@@ -59,4 +60,21 @@ public UserDto loadUserByUsername(String username) {
     }
 
 
+
+    @Override
+    public UserDto editUserDetails(String username,UserDto userDetails ) {
+            User userDetails2 = mapper.map(userDetails,User.class);
+
+            User oldUser = mapper.map(userRepository.findByUsername(username),User.class);
```

Comment:
No need for calling `mapper.map` here, right? `userRepository.findByUsername` already returns `User` object?

Reply:
solved